lasagne/layers/special.py

```diff
@@ -14,7 +16,9 @@
     "InverseLayer",
     "TransformerLayer",
     "ParametricRectifierLayer",
+    "RandomizedRectifierLayer",
     "prelu",
+    "rrelu",
```

Comment:
Minor thing, but please move `RandomizedRectifierLayer` one line down so it matches the order in the source code. Thanks!